knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -445,6 +462,19 @@ func updateEventingKafka(kafkachannel serverlessoperatorv1alpha1.Channel) mf.Tra
 	}
 }
 
+// setBootstrapServers sets Kafka bootstrapServers value in kafka-broker-config
+func setBrokerBootstrapServers(bootstrapServers string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() == "ConfigMap" && u.GetName() == "kafka-broker-config" {
+			log.Info("Found ConfigMap kafka-broker-config, updating it with bootstrapServers from spec")
+			if err := unstructured.SetNestedField(u.Object, bootstrapServers, "data", "bootstrap.servers"); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+}
```

Comment:
Since `bootstrapServers` might be invalid, should we add a check somewhere for it?

Reply:
For the channel we also do not really check it yet. but in a separate PR I could add some sort of check :man_shrugging: